onboarding_auth/onboardingAuthData/src/main/java/com/alexafit/onboardingauthdata/repository/OnboardingAuthRepositoryImpl.kt

```diff
@@ -0,0 +1,34 @@
+package com.alexafit.onboardingauthdata.repository
+
+import com.alexafit.onboardingauthdata.model.remote.Login
+import com.alexafit.onboardingauthdata.model.remote.LoginResponse
+import com.alexafit.onboardingauthdata.model.remote.Register
+import com.alexafit.onboardingauthdata.model.remote.RegisterResponse
+import com.alexafit.onboardingauthdata.remote.TaskyApi
+
+class OnboardingAuthRepositoryImpl(
+    private val taskyApi: TaskyApi
+) : OnboardingAuthRepository {
+    override suspend fun registerUser(registerBody: Register): Result<RegisterResponse> {
+        val registerResponse = taskyApi.registerUser(register = registerBody)
+        return if (registerResponse.isSuccessful) {
+            Result.success(RegisterResponse("Success"))
```

Comment:
Why do you need to return a "Success" as string?
Here you could just have the method return a 
Result<Unit>

And just return Result.success(Unit)
Or Result.failure(Throwable...)